benchmark/tpch.md

```diff
@@ -0,0 +1,90 @@
+---
+title: TiDB TPC-H 50G 性能测试报告 - v2.0
+category: benchmark
+draft: true
+---
+
+# TiDB TPC-H 50G 性能测试报告
+
+## 测试目的
+
+测试 TiDB 在 OLAP 场景下 1.0 和 2.0 版本的性能对比。
+
+> **注意**: 不同的测试环境可能使测试结果发生改变。
+
+## 测试环境
+
+### 1. 机器硬件信息
```

Comment:
To generate a more friendly PDF, usually do not use a number in the title level, because the PDF automatically generates another number for different levels of titles.